### PR TITLE
Support reading api keys from environment to help custom DC development

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/__init__.py [hunk 1]
```diff
@@ -277,13 +281,17 @@ def create_app():
...
     os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
 
-  if cfg.NEED_API_KEY:
-    # Only need to fetch the API key for local development.
-    secret_client = secretmanager.SecretManagerServiceClient()
-    secret_name = secret_client.secret_version_path(cfg.SECRET_PROJECT,
-                                                    'mixer-api-key', 'latest')
-    secret_response = secret_client.access_secret_version(name=secret_name)
-    app.config['DC_API_KEY'] = secret_response.payload.data.decode('UTF-8')
+  # Nseed to fetch the API key for non gcp environment.
```
Comment (chejennifer) [inline server/__init__.py:284]: Need

**Review by chejennifer — APPROVED** [review]
nice!

File: server/__init__.py (hunk 1 above)
Comment (shifucun) [reply server/__init__.py:284]: Done

**Comment by shifucun** [comment]
thanks for review!

**Pull request merged**